### PR TITLE
"add rowwise add backward op"

### LINKED ISSUE #3154: rowwise_add operator, Backward implementation.

### PR DESCRIPTION
**Pull request by dzhwinter**
fix #3154  rowwise backward

### CODE REVIEW

File: paddle/operators/rowwise_add_op.cc [hunk 1]
```diff
@@ -46,10 +46,25 @@ for i in xrange(X.shape[0]):
 )DOC");
   }
 };
+class RowWiseAddGradOp : public OperatorWithKernel {
```
Comment (wangkuiyi) [inline paddle/operators/rowwise_add_op.cc]: RowWise => Rowwise

File: paddle/operators/rowwise_add_op.h [hunk 2]
```diff
@@ -38,5 +38,24 @@ class RowWiseAddKernel : public OpKernel {
   }
 };
 
+template <typename Place, typename T>
+class RowWiseAddGradKernel : public OpKernel {
```
Comment (wangkuiyi) [inline paddle/operators/rowwise_add_op.h]: RowWise => Rowwise

File: paddle/operators/rowwise_add_op.cc (hunk 1 above)
Comment (dzhwinter) [inline paddle/operators/rowwise_add_op.cc]: Fix Done.

File: paddle/operators/rowwise_add_op.h (hunk 2 above)
Comment (dzhwinter) [inline paddle/operators/rowwise_add_op.h]: Done

File: python/paddle/v2/framework/tests/test_rowwise_add_op.py [hunk 3]
```diff
@@ -15,5 +16,17 @@ def setUp(self):
         self.outputs = {'Out': np.add(self.inputs['X'], self.inputs['b'])}
 
 
+class RowwiseAddGradOpTest(GradientChecker):
+    def test_rowwise_add(self):
+        op = create_op("rowwise_add")
+        inputs = {
+            "X": np.random.uniform(0.1, 1, [10, 10]).astype("float32"),
+            "b": np.random.uniform(0.1, 1, [10, 1]).astype("float32")
```
Comment (jacquesqiao) [inline python/paddle/v2/framework/tests/test_rowwise_add_op.py]: [10, 1] => [10]

**Pull request merged**